packages/validator/src/services/attestation.ts

```diff
@@ -225,7 +225,7 @@ export class AttestationService {
       ...(this.opts?.disableAttestationGrouping && {index: attestationNoCommittee.index}),
     };
     try {
-      (await this.api.beacon.submitPoolAttestations({signedAttestations})).assertOk();
+      (await this.api.beacon.submitPoolAttestationsV2({signedAttestations})).assertOk();
```

Comment:
we can do it like this for now but we need to some logic to use v2 api only after electra, otherwise this will break compatiblity if other clients if we merge this to unstable branch

Reply:
prefer to handle all forks now

Reply:
> we can do it like this for now but we need to some logic to use v2 api only after electra, otherwise this will break compatiblity if other clients if we merge this to unstable branch

I don't think we will merge this into unstable in the near future do we? My next step is to get `submitPoolAttestationsV2` to handle pre-electra. Then we can consider merging electra-branch into unstable

Reply:
Likely not, we need to think about a good strategy how we can get the electra branch into unstable safely without accidentally missing / reverting changes.

Reply:
then let's add `TODO - electra` to make sure we don't miss it